acceptance/python/src/test_basic.py

```diff
@@ -59,7 +59,7 @@ def test_basic():
 
 
     # Now she is ready to send it to the BigchainDB Network.
-    sent_transfer_tx = bdb.transactions.send(fulfilled_creation_tx, mode='commit')
+    sent_transfer_tx = bdb.transactions.send(fulfilled_creation_tx)
```

Comment:
The default mode is `async` so this will change the mode. Is that your intent?

Reply:
The default is actually [`commit`](https://github.com/bigchaindb/bigchaindb-driver/blob/fd088013e0159ff2baf89fabbb78f1c6df3a95ba/CHANGELOG.rst#050a2-2018-04-18).

Reply:
Really? That's weird, because if you don't include the `?mode=async` string at the end of the `POST /api/v1/transactions?mode=async` endpoint, then the default is `async`, not `commit`. In other words, the endpoint treats `async` as the default, so the Python driver should follow its lead.

Reply:
Oh, I see what you mean now. To me, there are many reasons why we should have `commit` as the default, also on the `POST /api/v1/transactions` endpoint. I'll open an issue for that.

Reply:
Okay, maybe post a link to that issue here in this discussion thread, for quick reference.

Reply:
I created an issue to discuss the default mode: issue #2307